apps/api/v2/src/modules/workflows/outputs/workflow.output.ts

```diff
@@ -0,0 +1,241 @@
+  message!: WorkflowMessageOutputDto;
+}
+
+export class WorkflowTriggerOffsetOutputDto {
+  @ApiProperty({ description: "Time value for offset", example: 24 })
+  @Expose()
+  value!: number;
+
+  @ApiProperty({
+    description: "Unit for the offset time",
+    example: HOUR,
+    enum: TIME_UNITS,
+  })
+  @Expose()
+  unit!: TimeUnitType;
+}
+
+export class WorkflowTriggerOutputDto {
+  @ApiProperty({
+    description: "Trigger type for the workflow",
+    example: BEFORE_EVENT,
+    enum: WORKFLOW_TRIGGER_TYPES,
+  })
+  @Expose()
+  type!: WorkflowTriggerType;
+
+  @ApiPropertyOptional({
+    description: "Offset details (present for BEFORE_EVENT/AFTER_EVENT)",
+    type: WorkflowTriggerOffsetOutputDto,
+  })
+  @Expose()
+  @ValidateNested()
+  @Type(() => WorkflowTriggerOffsetOutputDto)
+  offset?: WorkflowTriggerOffsetOutputDto;
+}
+
+export class WorkflowActivationOutputDto {
+  @ApiProperty({
+    description: "Whether the workflow is active for all event types associated with the team/user",
+    example: false,
+  })
+  @Expose()
+  isActiveOnAllEventTypes?: boolean = false;
+
+  @ApiPropertyOptional({
+    description: "List of Event Type IDs the workflow is specifically active on (if not active on all)",
+    example: [698191, 698192],
+  })
+  @Expose()
+  @IsArray()
+  activeOnEventTypeIds?: number[];
+}
+
+// --- Main Workflow Output DTO ---
+
+export class WorkflowOutput {
+  @ApiProperty({ description: "Unique identifier of the workflow", example: 101 })
+  @Expose()
+  id!: number;
+
+  @ApiProperty({ description: "Name of the workflow", example: "Platform Test Workflow" })
+  @Expose()
+  name!: string;
+
+  @ApiPropertyOptional({
+    description: "ID of the user who owns the workflow (if not team-owned)",
+    example: 2313,
+  })
+  @Expose()
+  userId?: number;
+
+  @ApiPropertyOptional({ description: "ID of the team owning the workflow", example: 4214321 })
+  @Expose()
+  teamId?: number;
+
+  @ApiProperty({ description: "Activation settings (scope)", type: WorkflowActivationOutputDto })
+  @Expose()
+  @ValidateNested()
+  @Type(() => WorkflowActivationOutputDto)
+  activation!: WorkflowActivationOutputDto;
+
+  @ApiProperty({ description: "Trigger configuration", type: WorkflowTriggerOutputDto })
+  @Expose()
+  @ValidateNested()
+  @Type(() => WorkflowTriggerOutputDto)
+  trigger!: WorkflowTriggerOutputDto;
+
+  @ApiProperty({ description: "Steps comprising the workflow", type: [WorkflowStepOutputDto] })
+  @Expose()
+  @IsArray()
+  @ValidateNested({ each: true })
+  @Type(() => WorkflowStepOutputDto)
+  steps!: WorkflowStepOutputDto[];
+
+  @ApiPropertyOptional({ description: "Timestamp of creation", example: "2024-05-12T10:00:00.000Z" })
+  @Expose()
+  createdAt?: Date | string;
+
+  @ApiPropertyOptional({ description: "Timestamp of last update", example: "2024-05-12T11:30:00.000Z" })
+  @Expose()
+  updatedAt?: Date | string;
+}
+
+// --- List Response Output DTO ---
+
+export class GetWorkflowsOutput {
```

Comment:
can we split in separate `get-workflows.output.ts` and `get-workflow.output.ts` to follow how we do it for rest of api v2.